features/rule_creation_protocol/tags_hierarchy.py

```diff
@@ -0,0 +1,5 @@
+tags_hierarchy = {
```

Comment:
Is it possible to move this to the data-model repo? We need to have access to this in the score card repo as well.

Reply:
Sure :) 
I've added [these to the data-model repo](https://github.com/buildingSMART/ifc-validation-data-model/pull/21/) .
I thought we could probably also use more info regarding the functional parts to colour the blocks (is also added in the same PR)